code/src/java/gmgen/plugin/PcgCombatant.java

```diff
@@ -896,7 +898,11 @@ protected String getStatBlockLineSpells()
 				 <br>
 				 <!-- Start Racial Innate Spell listing -->
 				 |FOR,%spell,0,COUNT[SPELLSINBOOK%class.%spellbook.%level]-1,1,0|
-				 <a href="spell:|SPELLMEM.%class.%spellbook.%level.%spell.NAME|\|SPELLMEM.%class.%spellbook.%level.%spell.DESC|\|SPELLMEM.%class.%spellbook.%level.%spell.RANGE|\|SPELLMEM.%class.%spellbook.%level.%spell.CASTINGTIME|\|SPELLMEM.%class.%spellbook.%level.%spell.SAVEINFO|\|SPELLMEM.%class.%spellbook.%level.%spell.DURATION|\|SPELLMEM.%class.%spellbook.%level.%spell.TARGET|" class="dialog">
+				 <a href="spell:|SPELLMEM.%class.%spellbook.%level.%spell.NAME|\|
+				    SPELLMEM.%class.%spellbook.%level.%spell.DESC|\|SPELLMEM.%class.%spellbook.%level.%spell.RANGE|
+				    \|SPELLMEM.%class.%spellbook.%level.%spell.CASTINGTIME|\|SPELLMEM.%class.%spellbook.%level.%spell
+				    .SAVEINFO|\|SPELLMEM.%class.%spellbook.%level.%spell.DURATION|\|SPELLMEM.%class.%spellbook.%level
+				    .%spell.TARGET|" class="dialog">
```

Comment:
This is likely to break output because you are wrapping items going through the output system.